webpack/assets/javascripts/react_app/Root/ReactApp.js

```diff
@@ -8,6 +8,7 @@ import Layout, { propTypes as LayoutPropTypes } from '../components/Layout';
 import AppSwitcher from '../routes';
 
 import apolloClient from './apollo';
+import ToastsList from '../components/ToastsList';
 
 const ReactApp = ({ layout, metadata, toasts }) => {
   const contextData = { metadata, toasts };
```

Comment:
Do we still need the toasts under the foreman context?

Reply:
I was also thinking about removing it, though I was too afraid to break plugins...
I guess we could first remove the usage of useForemanContext with plugins and then remove it as well.

Reply:
How about a deprecation warning?

Reply:
Seems that there's no helper method specific to toasts, so it would be kinda hard to deprecate,
This is what we have in `ForemanContext.js`

```js
export const useForemanContext = () => React.useContext(getForemanContext());

const useForemanMetadata = () => useForemanContext().metadata;

export const useForemanVersion = () => useForemanMetadata().version;
export const useForemanSettings = () => useForemanMetadata().UISettings;
export const useForemanDocUrl = () => useForemanMetadata().docUrl;
export const useForemanOrganization = () => useForemanMetadata().organization;
export const useForemanLocation = () => useForemanMetadata().location;
export const useForemanUser = () => useForemanMetadata().user;
```

Added a subtask of the issue I opened in Redmine: https://projects.theforeman.org/issues/32871
so we could better tackle this across plugins.

Reply:
Looks like just [webhooks plugin](https://github.com/theforeman/foreman_webhooks/blob/1aa674a4e6dd37a37b2e93e2d77d4a6ffb8cb071/webpack/ForemanWebhooks/Routes/Webhooks/WebhooksIndexPage/index.js#L42) consumes toasts from `ForemanContext`, worth a quick fix

Reply:
nice catch @amirfefer! opened https://github.com/theforeman/foreman_webhooks/pull/39

Reply:
We can drop it from the context in a follow-up pr, right now this shouldn't be a breaking change afaict.

Reply:
actually, if we don't include it in the context (or maybe include an empty array instead of the actual toasts), the webhooks plugin should function just fine until it's cleaned up as it won't be displaying any toasts on its own and we won't get duplicates